expr/socket.go

```diff
@@ -0,0 +1,73 @@
+package expr
```

Comment:
Please copy the license header from the other files to the top of this file.

Reply:
I change 2018 to 2023.